src/sqlancer/common/query/SQLQueryError.java

```diff
@@ -0,0 +1,116 @@
+package sqlancer.common.query;
+
+public class SQLQueryError implements Comparable<SQLQueryError> {
```

Comment:
We could try to merge this quickly by creating a separate PR for this and adding also some tests for these methods.